rebar.config

```diff
@@ -0,0 +1,52 @@
+%%% -*- mode: erlang; -*-
+{erl_opts,
+ [
+  warn_unused_vars,
+  warn_export_all,
+  warn_shadow_vars,
+  warn_unused_import,
+  warn_unused_function,
+  warn_bif_clash,
+  warn_unused_record,
+  warn_deprecated_function,
+  warn_obsolete_guard,
+  strict_validation,
+  warn_export_vars,
+  warn_exported_vars,
+  warn_missing_spec,
+  warn_untyped_record,
+  debug_info,
+  {d, rebar}
+ ]
+}.
+
+{erl_first_files, ["src/ktn_recipe.erl"]}.
+
+{deps,
+ [
```

Comment:
I'm not sure I understand `rebar3` completely, but there are some deps that katana needs to run (i.e. those are not just _test deps_). In particular `inaka_aleppo` `xref_runner` and `elvis_core` (which BTW is a circular dependency and that's _fixed_ in erlang.mk-land using `IGNORE_DEPS += katana`).
Shouldn't those deps be listed here?